src/vue/pages/Trade.vue

```diff
@@ -0,0 +1,257 @@
+<template>
+  <div class="trade">
+    <top-bar>
```

Comment:
got an idea of how to get rid of transmit event from parent to the child. let’s extract this topbar and drawer components to a new component like TradeTopBar and just reuse on all of the trade screens. the TopBar will emit the `reload-order-list` event the component will catch it and do the thing, we don’t need Bus here anymore. Trade.vue by itself will be just empty page that use router-view only